### DISCUSSION TITLE
Add Tag Feature for md bolg

### DISCUSSION THREAD

**Discussion by starryskyhk** [withastro/starlight#1382] [Feature Requests]
### What version of `starlight` are you using?

latest

### What is your idea?

I think I can add a tag list to the navigation bar and add tag identifiers to the md file to filter files by tags.

### Why is this feature necessary?

Can help us better distinguish blogs

### Participation

- [x] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Hi @starryskyhk 👋 

Is this comment intended to be for https://github.com/HiDeoo/starlight-blog?

**Reply by starryskyhk** [reply]
sorry, I don't know this repo. Maybe it has completed the tag feather. I will take a look.

**Comment by dragomano** [comment]
I need translatable tags for doc pages, but I don't need a blog. Is it possible?

**Comment by HiDeoo** [comment]
As the initial question was probably for the blog plugin, I'm going to close this discussion for now.

Regarding the additional question about translatable tags for doc pages, I am not aware of any existing solution for that but I'm assuming that could be part of a small plugin.
